command/server.go

```diff
@@ -1992,6 +2001,18 @@ CLUSTER_SYNTHESIS_COMPLETE:
 	return retCode
 }
 
+func (c *ServerCommand) notifySystemd(status string) {
+	sent, err := systemd.SdNotify(false, status)
+	if err != nil {
+		c.logger.Error("error notifying systemd", "error", err)
+	}
+	if sent {
+		c.logger.Debug("sent systemd notification", "notification", status)
+	} else {
+		c.logger.Debug("would have sent systemd notification (systemd not present)", "notification", status)
```

Comment:
these are likely very infrequent, but why do we want this log?

Reply:
also i think this can be inaccurate if `err != nil` - I looked at `SdNotify` and it'll return `false` with a non-nil error so this log might be confusing?

Reply:
The logging was partially to assist implementation, but could also help people debugging since it requires correct configuration in systemd to work as well.   You're right that the if sent/else block should only happen if err is nil.